src/main/java/org/sagebionetworks/web/client/widget/upload/FileHandleUploadWidgetImpl.java

```diff
@@ -26,6 +29,7 @@ public FileHandleUploadWidgetImpl(FileHandleUploadView view, MultipartUploader m
 		this.multipartUploader = multipartUploader;
 		this.synapseJsniUtils = synapseJsniUtils;
 		this.view.setPresenter(this);
+		this.view.allowMultipleFileUpload(true);

```

Comment:
Default to false, but expose to others so the Profile Validation workflow can set to true.